website/docs/cloud-docs/integrations/run-tasks/index.mdx

```diff
@@ -91,7 +91,9 @@ The HCP Packer validation run task checks the image artifacts within a Terraform
 
 To get started, [create an HCP Packer account](https://cloud.hashicorp.com/products/packer) and follow the instructions in the [HCP Packer Run Task](https://cloud.hashicorp.com/docs/packer/manage-image-use/terraform-cloud-run-tasks) documentation.
 
-## Run Tasks Technology Partners
+## Available Run Tasks
+
+-> **Note:** You can view the most up-to-date list of run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks).
```

Comment:
This is great. It forced me to ask one question: if the registry is the most up-to-date list of Run Tasks, why does the **Run Tasks Technology Partners** section still exist? Perhaps it would be best to remove this entirely so we do not have an out-of-date list of integrations.

Reply:
That's a great point, Ryan, we are planning to remove it within a month (Nov 22) since it was the go-to place we were using to point all our partners we're just letting them know the new location and will remove it

Reply:
in other words, Run Tasks Technology Partners section will act as a stop-gap for a while with a banner saying "The most up-to-date list is available on the registry" and then we'll remove it after a month

Reply:
@hcrhall Are you okay with this stop-gap solution?